.travis.yml

```diff
@@ -2,15 +2,16 @@ language: node_js
 node_js:
   - node
 cache: npm
-
 install:
   - npm ci
-
 script:
   - npm test
-
 deploy:
-  skip-cleanup: true
```

Comment:
Why was the skipping of cleanup discarded?

Reply:
skip-cleanup as been deprecated by Travis its just cleanup now, I think I added it but now I see, I should have set it to false based on the skip-cleanup.